code/sorting/src/cocktail_sort/cocktail_sort.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <conio.h>
```

Comment:
Don't use this header. Reformat both your files to use clang-format with 4 spaces.

Reply:
can you please tell me what i need to do exactly?